src/annotator/integrations/html-side-by-side.js

```diff
@@ -145,24 +163,16 @@ function* textNodesInRect(root, rect, shouldVisit = () => true) {
  * Find content within an element to use as an anchor when applying a layout
  * change to the document.
  *
- * @param {Element} scrollRoot
+ * @param {Element} root
```

Comment:
Would be helpful to expand this to explain what "root" means.

Reply:
I think this should be covered by the comment above ("Find content within an element...") since there is only one `Element` argument to the function.